runtime-services-modeling/decisions-with-dmn.adoc

```diff
@@ -653,12 +661,91 @@ image::dmn-list-expression-example.png[]
 
 The following boxed list expression also identifies approved credit score agencies but uses FEEL logic to define the agency status (Inc., LLC, SA, GA) based on a DMN input node:
 
+[id="approved-credit-score-agency-list-example"]
 .Boxed list expression using FEEL logic for approved credit score agency status
 image::dmn-list-expression-example2.png[]
 
 .DMN diagram for approved credit score agency status
 image::dmn-list-expression-example2a.png[]
 
+[id="con-dmn-conditional-expressions"]
+==== Boxed conditional expressions
+
+[role="_abstract"]
+A boxed conditional offers a visual representation of an `if` statement using three rows.
+The expression in the `if` part must resolve to a boolean.
+When the `if` resolves to a `true`, `then` part is returned.
+When the `if` resolves to a `false`, `else` part is returned.
+
+.Conditional expression of MonthlyFee computation
+image::runtime-services-modeling/dmn-conditional-expression.png[]
+
+NOTE: This feature was introduced as part of the {PRODUCT} 9.1 and DMN 1.5 releases.
+
+[id="con-dmn-iterator-expressions"]
+==== Boxed iterator expressions
+
+[role="_abstract"]
+A boxed iterator offers a visual representation of an iterator statement.
+For the `for` loop, the right part of the `for` displays the iterator variable name.
+The second row holds an expression representing the collection that will be iterated over.
+The expression in the `in` row must resolve to a collection.
+The last row contains the expression that will process each element of the collection.
+
+The example below is an alternative way of producing list similar as <<approved-credit-score-agency-list-example>>.
+
+.Iterator expression of Approved credit score agencies
+image::runtime-services-modeling/dmn-iterator-expression.png[]
+
+NOTE: This feature was introduced as part of the {PRODUCT} 9.1 and DMN 1.5 releases.
+
+[id="con-dmn-every-expressions"]
+==== Boxed every expressions
+
+[role="_abstract"]
+A boxed iterator offers a visual representation of an iterator statement.
```

Comment:
It has the same text of the `Boxed iterator expressions` section, maybe changing the previous section to `Boxed for expression`? And be more specific on the paragraph as this is related to the `every` expression.

`A boxed every offers a visual representation of an iterator statement`

Reply:
done